v0/destinations/facebook_pixel/transform.js

```diff
@@ -244,10 +278,19 @@ const responseBuilderSimple = (
       category.standard = true;
     }
     if (Object.keys(customData).length === 0 && category.standard) {
-      throw new CustomError(
-        "No properties for the event so the event cannot be sent.",
-        400
-      );
+      throw new ErrorBuilder()
+        .setMessage("No properties for the event so the event cannot be sent.")
```

Comment:
```suggestion
        .setMessage("No valid fields are present in 'properties' for a standard event")
```

Reply:
I wonder, What do valid fields represent here ?

Reply:
We can only define exclusion fields here. How about this?

```
"No valid fields are present in 'properties' for a standard event. The excluded fields are <exclusion_fields_list>"
```

Reply:
Not sure, if this is right but based on your comments making the changes. Please check and let me know if it is ok

Reply:
"After excluding <exclusion_fields_list_here>, no fields are present in 'properties' for a standard event" ?

Reply:
Changed it. Let me know if it is ok